FirebaseVertexAI/Tests/Unit/ChatTests.swift

```diff
@@ -46,7 +47,13 @@ final class ChatTests: XCTestCase {
       return (response, fileURL.lines)
     }
 
-    let model = GenerativeModel(name: "my-model", apiKey: "API_KEY", urlSession: urlSession)
+    let model = GenerativeModel(
+      name: "my-model",
+      apiKey: "API_KEY",
+      requestOptions: RequestOptions(),
+      appCheck: nil,
```

Comment:
Is it feasible/worthwhile to do any unit tests with a non-nil appCheck?

Reply:
I think it's feasible and definitely worthwhile to add. I'll do that in a follow-up but I think it would be good to check the App Check headers (verifying that they're not there in the current cases and have a value when an App Check instance is provided).

Reply:
Added in https://github.com/firebase/firebase-ios-sdk/pull/12590.